sw/airborne/firmwares/fixedwing/stabilization/stabilization_adaptive.c

```diff
@@ -489,6 +553,17 @@ inline static void h_ctl_pitch_loop(void)
   h_ctl_ref.pitch_accel = 0.;
 #endif
 
+#ifdef USE_KFF_UPDATE
```

Comment:
just started to experiment with it, at least it is not doing any harm so far. deftly need more testing on this. but I noticed that it was only implemented for roll so added it for pitch as well.
you are not using it at all? any drawbacks from you're side?
maybe we can separate both loops one for roll and one for pitch? two flags, so everyone can choose if he wants to use both or just one...